lib/moonshot/yaml_stack_template.rb

```diff
@@ -0,0 +1,26 @@
+require 'yaml'
+require_relative 'stack_template'
+
+module Moonshot
+  # Handles YAML formatted AWS template files.
+  class YamlStackTemplate < StackTemplate
+    def initialize(filename, log:)
+      super(filename, log: log)
+      @body = YAML.load_file(@filename) if exists?
+    end
+
+    def parameters
+      @body['Parameters'].map do |k, v|
```

Comment:
Same comment as the resource_names method below -- which leads me to believe you have a reason for not using `#fetch` in this class?